modules/meeting/app/contracts/meeting_agenda_items/create_contract.rb

```diff
@@ -38,6 +38,7 @@ def self.assignable_meetings(user)
       Meeting
         .open
         .or(Meeting.in_progress)
+        .or(Meeting.draft)
```

Comment:
```suggestion
        .not_closed
```

I think this would be more robust for future changes?